src/charm.py

```diff
@@ -93,6 +93,9 @@ def __init__(self, *args):
                 # See https://github.com/canonical/charmed-kafka-snap for details
                 {"path": "/metrics", "port": f"{JMX_EXPORTER_PORT}"},
             ],
+            metrics_rules_dir="./src/alert_rules/prometheus",
+            logs_rules_dir="./src/alert_rules/loki",
+            log_slots=["charmed-kafka:logs"],
```

Comment:
**nitpick/todo:** Could these be `literals`?

Or better, have `log_slots` added as class vars to `KafkaSnap`, and the others go to `literals.py`?

Reply:
Will do in a following PR